pkg/operator/ceph/object/objectstore.go

```diff
@@ -1000,3 +999,38 @@ func errorOrIsNotFound(err error, msg string, args ...string) error {
 	}
 	return errors.Wrapf(err, msg, args)
 }
+
+// check whether zone endpoint list need to be updated or not
+func UpdateZoneEndpointList(zones []zoneType, desiredEndpointList []string, zoneName string) bool {
```

Comment:
For an exported function (starting with a capital letter), the doc comment should start with the name of the function.

```go
// UpdateZoneEndpointList checks whether the zone endpoint list need to be updated or not
func UpdateZoneEndpointList(zones []zoneType, desiredEndpointList []string, zoneName string) bool {
```